### PR TITLE
[DOCs/operators]: Minor syntax edits of Release flow and Sandbox pages

### PR DESCRIPTION
**Pull request by serinko**
A few mistakes slipped under Friday pressure - here is the fix.

### CODE REVIEW

**Comment by serinko** [comment]
Ready for review on my side.

File: documentation/operators/src/sandbox.md [hunk 1]
```diff
@@ -10,21 +12,25 @@ This page is for Nym node operators. If you want to run NymVPN CLI over Sandbox
...
 ```sh
-# this example is for mixnode mode
+# this example is for nym-node in mixnode mode
 ./nym-node --config-env-file <PATH/TO/>sandbox.env run --mode mixnode
 
-# this example is for exit-gateway mode
+# this example is for nym-node in exit-gateway mode
 ./nym-node --config-file-env <PATH/TO/>sandbox.env run --mode exit-gateway --id <ID> --public-ips "$(curl -4 https://ifconfig.me)" --hostname "<YOUR_DOMAIN>" --http-bind-address 0.0.0.0:8080 --mixnet-bind-address 0.0.0.0:1789 true --location <COUNTRY_FULL_NAME>
+
+# In case you downloaded sandvbox.env to the same directory, <PATH> is not needed
```
Comment (tommyv1987) [inline documentation/operators/src/sandbox.md]: typo here `sandbox`

File: documentation/operators/src/sandbox.md [hunk 2]
```diff
@@ -10,21 +12,25 @@ This page is for Nym node operators. If you want to run NymVPN CLI over Sandbox
 
 > Any syntax in `<>` brackets is a user's unique variable. Exchange with a corresponding name without the `<>` brackets.
 
-To run Nym binaries in Sandbox Testnet you need to get the `.env` configuration file and point your binary to it. Follow the steps below:
+To run Nym binaries in Sandbox Testnet you need to get `sandbox.env` configuration file and point your binary to it. Follow the steps below:
```
Comment (tommyv1987) [inline documentation/operators/src/sandbox.md]: To run Nym binaries in the sandbox testnet, you need to get the sandbox.env configuration file and point your binary to it `the inclusion of the comma after testnet`

File: documentation/operators/src/sandbox.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Sandbox Testnet
 
-Nym node operators can run their nodes on Nym Sandbox testnet environment. Whether it's testing new configuration, hot features from Nym developers or just trying to setup a node for the first time, this environment is for you. Below are steps to [setup your environment](#sandbox-environment-setup) and an introduction to [Sandbox token faucet](#sandbox-token-faucet).
+Nym node operators can run their nodes in Nym Sandbox testnet environment. Whether it's testing new configuration, hot features from Nym developers or just trying to setup a node for the first time, this environment is for you.
```
Comment (tommyv1987) [inline documentation/operators/src/sandbox.md:3]: Liking the sales pitch tone here :)

**Review by tommyv1987 — COMMENTED** [review]
Left some notes.

File: documentation/operators/src/release-cycle.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Release Cycle
 
-Nym operators community grows in quality and quantity. With node operators and developers joining the efforts to build Mixnet more robust and scalable, testing new features, sharing integration pull requests and generally taking an active part in Nym development, more transparency on the release cycle is required.
+Nym operators community grows in quality and quantity. With node operators and developers joining the effort to build Mixnet more robust and scalable, testing new features, sharing integration pull requests and generally taking an active part in Nym development, more transparency on the release cycle is required.
```
Comment (mfahampshire) [inline documentation/operators/src/release-cycle.md]: Small grammatical changes: 

`The Nym operator community is growing in quality and quantity. With node operators and developers joining the effort to make the Mixnet more robust and scalable, testing new features, sharing integration pull requests and generally taking an active part in Nym development, more transparency on the release cycle is required.`

File: documentation/operators/src/sandbox.md [hunk 5]
```diff
@@ -45,6 +51,6 @@ export NYMNODE_CONFIG_ENV_FILE_ARG=<PATH/TO/sandbox.env>
 
 ## Sandbox Token Faucet
 
-To run your nodes in the sandbox environment, you need testnet version of NYM token, that can be aquired at [faucet.nymtech.net](https://faucet.nymtech.net).
+To run your nodes in Sandbox environment, you need testnet version of NYM token, that can be aquired from [faucet.nymtech.net](https://faucet.nymtech.net).
```
Comment (mfahampshire) [inline documentation/operators/src/sandbox.md:53]: We should pick on whether to always capitalise `Sandbox` or not - don't mind either way, just for consistency.

**Review by mfahampshire — CHANGES_REQUESTED** [review]
Minor grammatical changes requested - otherwise LGTM :)

File: documentation/operators/src/sandbox.md (hunk 5 above)
Comment (serinko) [reply documentation/operators/src/sandbox.md:53]: I did before this note - so let ***S***andbox be it!

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**